Adding SQLServer schema collection information (#26127)
DataDog/documentation: layouts/shortcodes/dbm-sqlserver-agent-config-examples.en.md

      history_row_limit: 10000
```

### Collecting schemas

> To enable this feature, use the `schemas_collection` option. Schemas are collected on databases for which the Agent has `CONNECT` access.

Reviewer: Can we add min required Agent version here?

> To enable this feature, use the `schemas_collection` option. Schemas will be collected on databases for which the Agent has `CONNECT` access.

Reviewer 2: ```suggestion
To enable this feature, use the `schemas_collection` option. Schemas are collected on databases for which the Agent has `CONNECT` access.
```
